### ISSUE TITLE
SDK server fails to run if an optional data binding has no default value

### ISSUE THREAD

**Issue by robinhouston** [kiln/flourish-sdk#23]
Release 2.8.0 added support for optional data bindings, which can be enabled by adding `optional: true` to the definition of a data binding that has `type: column`. Optional data bindings need not have a default value: if they do not, the default is blank.

The SDK server fails to start on a template that has an optional data binding with no default value.

**Comment by annapowellsmith** [comment]
For reference/searchability the error it fails with is 

```
 Build script finished
  Running server
😱  Failed to start server
😱  Cannot read property 'indexOf' of null
```

**Comment by micycle** [comment]
Should be fixed by https://github.com/kiln/flourish/pull/506

**Comment by robinhouston** [comment]
This will be fixed in 2.9.0.
